packages/dev/loaders/src/glTF/glTFFileLoader.ts

```diff
@@ -309,6 +310,29 @@ export class GLTFFileLoader implements IDisposable, ISceneLoaderPluginAsync, ISc
         this._onMeshLoadedObserver = this.onMeshLoadedObservable.add(callback);
     }
 
+    /**
+     * Callback raised when the loader creates a skin after parsing the glTF properties of the skin node.
+     * @see https://doc.babylonjs.com/divingDeeper/importers/glTF/glTFSkinning#ignoring-the-transform-of-the-skinned-mesh
+     * @param node - the transform node that corresponds to the original glTF skin node used for animations
+     * @param skinnedNode - the transform node that is the skinned mesh itself or the parent of the skinned meshes
+     */
+    public readonly onSkinLoadedObservable = new Observable<{ node: TransformNode; skinnedNode: TransformNode }>();
+
+    private _onSkinLoadedObserver: Nullable<Observer<{ node: TransformNode; skinnedNode: TransformNode }>>;
+
+    /**
+     * Callback raised when the loader creates a mesh after parsing the glTF properties of the mesh.
+     * Note that the callback is called as soon as the mesh object is created, meaning some data may not have been setup yet for this mesh (vertex data, morph targets, material, ...)
+     */
+    public set onSkinLoaded(callback: (node: TransformNode, skinnedNode: TransformNode) => void) {

```

Comment:
why is this needed? (this and the observer)

Reply:
Removed